packages/ra-core/src/form/choices/ChoicesContext.ts

```diff
@@ -24,7 +25,7 @@ export type ChoicesContextValue<RecordType extends RaRecord = any> = {
     isLoading: boolean;
     page: number;
     perPage: number;
-    refetch: () => void;
+    refetch: (() => void) | UseGetListHookValue<RecordType>['refetch'];
```

Comment:
Why `UseGetListHookValue`?

Reply:
Idem -> Done to  continue the compatibility between `ListContext` and `ChoicesContext` for `DatagridInput`